### DISCUSSION TITLE
Upload Data from .xls file into DB

### DISCUSSION THREAD

**Discussion by patkeiman95** [dbeaver/dbeaver#11187] [Q&A] [answered]
Hello together, I have a large .xls file which I would like to upload into the MariaDB Database. I created a table with corresponding columns but don’t know how I can upload the data into it. Could anyone give me some advice please?

Thank you.

**Comment by tati-kru** [comment, marked as answer]
You can find the information about the data import here: https://github.com/dbeaver/dbeaver/wiki/Data-transfer.

Please, draw your attention that import from .xls is available in DBeaver EE only. If you work with DBeaver CE you can use .csv files for the import.
